test/src/main/java/org/apache/accumulo/test/functional/AccumuloClientIT.java

```diff
@@ -46,11 +46,15 @@
 import org.apache.accumulo.harness.AccumuloClusterHarness;
 import org.junit.After;
 import org.junit.Test;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
 
 import com.google.common.collect.Iterables;
 
 public class AccumuloClientIT extends AccumuloClusterHarness {
 
+  Logger log = LoggerFactory.getLogger(AccumuloClientIT.class);
```

Comment:
This seems to be a new logger, but I don't see any new log messages in this PR. Is this unused?

Reply:
Dropped.